apps/speed.c

```diff
@@ -1272,9 +1225,6 @@ static int run_benchmark(int async_jobs, int (*loop_function)(void *), loopargs_
                         total_op_count += job_op_count;
                     }
                     --num_inprogress;
-#if defined(OPENSSL_SYS_UNIX)
-                    FD_CLR(job_fd, &waitfdset);
```

Comment:
The waitfdset is now refreshed at every iteration so there's no need to explicitly clear the fd when the job has completed.